Makefile.am

```diff
@@ -5,7 +5,6 @@ all:
 	$(MAKE) -C cppForSwig
 
 clean-local:
-	$(MAKE) -C cppForSwig clean
```

Comment:
With autotools, clean-local is for extra cleaning that autotools doesn't do by default. When I was originally creating Makefile.am, I didn't realize that autotools automatically picked up on the fact that cppForSwig has to be cleaned.

So if that is present under the clean-local rule, it gets run twice in total.

Normally that doesn't matter, but the Debian build tools exit with an error if you try to rm files that don't exist. So the second invocation of clean on cppForSwig causes an error. So I had to remove this extraneous clean to get the deb package building.

So that is why I am sneaking it in with the deb stuff.